kadmin/rpc.c

```diff
@@ -1003,6 +1003,7 @@ process_stream(krb5_context contextp,
 		krb5_errx(contextp, 1, "gss error, exit");
 	    }
 	    if ((maj_stat & GSS_S_CONTINUE_NEEDED) == 0) {
+                extern char *local_kdc_name;
```

Comment:
Build is failing on nested extern, can we put this in a header?

```
/home/runner/work/heimdal/heimdal/kadmin/rpc.c: In function ‘process_stream’:
[2821](https://github.com/heimdal/heimdal/actions/runs/3906574195/jobs/6674894421#step:4:2822)
/home/runner/work/heimdal/heimdal/kadmin/rpc.c:1006:30: error: nested extern declaration of ‘local_kdc_name’ [-Werror=nested-externs]
[2822](https://github.com/heimdal/heimdal/actions/runs/3906574195/jobs/6674894421#step:4:2823)
                 extern char *local_kdc_name;
[2823](https://github.com/heimdal/heimdal/actions/runs/3906574195/jobs/6674894421#step:4:2824)
```

Reply:
I'll fix it.  We don't always bother with headers for externs like this.

Reply:
Sure, no stylistic objections, just doesn't build with current buildbot config